e2e/content-services/metadata/metadata-permissions.e2e.ts

```diff
@@ -35,13 +35,9 @@ describe('permissions', () => {
     const METADATA = {
```

Comment:
Do we need that whole object? Only PROPERTY_TAB is used so we can have just single const string with that one value